velox/functions/prestosql/benchmarks/JsonExprBenchmark.cpp

```diff
@@ -189,8 +189,6 @@ class JsonBenchmark : public velox::functions::test::FunctionBenchmarkBase {
         {"folly_json_array_length"});
     registerFunction<SIMDJsonArrayLengthFunction, int64_t, Json>(
         {"simd_json_array_length"});
-    registerFunction<JsonExtractScalarFunction, Varchar, Json, Varchar>(
```

Comment:
@mbasmanova, I note you have moved `JsonExtractScalarFunction` to this file. Do I need to put the deleted code back for this file?

Reply:
@mbasmanova, just reverted. Thanks!